test/e2e/flowaggregator_test.go

```diff
@@ -168,6 +168,10 @@ type testFlow struct {
 	checkDstSvc bool
 }
 
+type Response struct {
```

Comment:
let's rename it to something more precise, e.g. `IPFIXCollectorResponse`

Reply:
Fixed. Thanks.